extension/src/utils.js

```diff
@@ -0,0 +1,16 @@
+// in the polkadotAPIs Object we will store all the different Polkadot APIs to sign in extension
```

Comment:
why not use this utils function also in the app?

Reply:
true. It could be called from the connection module. Let me try that

Reply:
OK, so what I did in the end is to have the `polkadotAPIs` object in state instead of the `polkadotAPI` one. Much better. Only it gets all erased with refresh, but logically happens the same for all objects.

Still, the performance improvement is quite significant. So let's merge!!

Reply:
ur still duplicating code here. please use this utils function also in the app.